fedbiomed/transport/client.py

```diff
@@ -343,21 +344,108 @@ def __init__(self, channels: Channels) -> None:
         """
         self._channels = channels
 
+        self.task_name = "generic listener"
+        self._want_retry_grpc_error = False
```

Comment:
The name `_retry_on_error` may better fit this attribute.

Reply:
OK, will change it as you recommend